frontend/src/containers/Review/ReviewDetail/ReviewDetail.js

```diff
@@ -117,7 +123,7 @@ class ReviewDetail extends Component {
                                     ) : null}
                                 </div>
                                 <Form className="new-reply">
-                                    <Form.Label className="username">Username </Form.Label>
+                                    <Form.Label className="username">Girin </Form.Label>
```

Comment:
I think it would be better that we choose the master version here.

Reply:
ReviewDetail and ReviewControl would be changed after merging backend api for reviews. So now, it is not important whether it is Grin or Usrename(both just text), since it would be modified to {userName} soon.